be/src/vec/functions/function_string.h

```diff
@@ -581,7 +588,22 @@ class FunctionLeft : public IFunction {
         temp_arguments[0] = arguments[0];
         temp_arguments[1] = num_columns_without_result;
         temp_arguments[2] = arguments[1];
-        SubstringUtil::substring_execute(block, temp_arguments, result, input_rows_count);
+
+        bool is_nullable {};
```

Comment:
it will be more clear if change to
```suggestion
        bool is_nullable = nullptr;
```